src/styles/css/index.css

```diff
@@ -1,22 +0,0 @@
-body {
-  margin: 0;
-  padding: 0; }
-
-.side_menu {
```

Comment:
I see you removed the side menu styles from this file like Peter requested, but it doesn't look like you included another css file with these styles?